test/utilities/windUtilities.spec.js

```diff
@@ -0,0 +1,14 @@
+import ava from 'ava';
+
+import { getNewWind } from '../../src/assets/scripts/client/utilities/windUtilities';
+
+const windMock = {
+    speed: 10,
+    angle: 400
+};
+
+ava('.getNewWind() does not return a value greater than 360', (t) => {
```

Comment:
you should add a test case for the other logical branch branch; when this function is passed values greater than 360